x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_fetch_browser_fields_capabilities.tsx

```diff
@@ -49,13 +49,7 @@ export const useFetchBrowserFieldCapabilities = ({
   }, [featureIds, http, toasts]);
 
   useEffect(() => {
-    if (featureIds.includes(INVALID_FEATURE_ID)) {
-      setIsLoading(false);
-    }
-  }, [featureIds]);
-
-  useEffect(() => {
-    if (isLoading !== undefined) return;
+    if (isLoading !== undefined || featureIds.includes(INVALID_FEATURE_ID)) return;
```

Comment:
we are doing `isLoading !== undefined` because we want to make sure we are only fetching the date one times. if that's the case I will write a unit test for that.